actions/local_storage.jsx

```diff
@@ -21,14 +19,11 @@ export function setPreviousTeamId(teamId) {
     };
 }
 
-export function setPreviousRecentEmojis() {
+export function setRecentEmojis(recentEmojis = []) {
     return (dispatch, getState) => {
         const currentUserId = getCurrentUserId(getState());
 
-        const recentEmojis = getState().storage.storage[Constants.RECENT_EMOJI_KEY];
-        if (recentEmojis && recentEmojis.value) {
-            LocalStorageStore.setPreviousRecentEmojis(currentUserId, recentEmojis.value);
-        }
+        LocalStorageStore.setRecentEmojis(currentUserId, recentEmojis.join(','));
```

Comment:
Set and get it as string instead of array to avoid `JSON.stringify` or `JSON.parse`, respectively.

Reply:
Agree with not doing this in the caller -- the encoding is an implementation detail -- but unclear why we wouldn't still want to do it internally vs. a manual encoding of commas. I guess commas aren't part of the valid emoji character set? Seems like it might be safer to still encode via JSON internally here for future robustness.

Reply:
Yes, that makes a lot of sense.  Thanks @lieut-data, no updated.  Please take a look again.